lib/containers/urls.pm

```diff
@@ -51,64 +51,211 @@ sub get_opensuse_registry_prefix {
+            },
+            available_arch => ['x86_64', 'ppc64le', 's390x']
+        },
+        '15-SP1' => {
+            released_registry => sub { "registry.suse.com/suse/sle15:15.1" },
+            totest_registry   => sub {
+                "registry.suse.de/suse/sle-15-sp1/update/cr/totest/images/suse/sle15:15.1";
+            },
+            available_arch => ['x86_64', 'aarch64', 'ppc64le', 's390x']
+        },
+        '15-SP2' => {
+            released_registry => sub { "registry.suse.com/suse/sle15:15.2" },
+            totest_registry   => sub {
+                "registry.suse.de/suse/sle-15-sp2/update/cr/totest/images/suse/sle15:15.2";
+            },
+            available_arch => ['x86_64', 'aarch64', 'ppc64le', 's390x']
+        },
+        '15-SP3' => {
+            released_registry => sub { "registry.suse.com/suse/sle15:15.3" },
+            totest_registry   => sub {
+                "registry.suse.de/suse/sle-15-sp3/update/cr/totest/images/suse/sle15:15.3";
+            },
+            available_arch => ['x86_64', 'aarch64', 'ppc64le', 's390x']
+        },
+        '15-SP4' => {
+            released_registry => sub { '' },
+            totest_registry   => sub {
+                "registry.suse.de/suse/sle-15-sp4/ga/test/images/suse/sle15:15.4";
+            },
+            available_arch => ['x86_64', 'aarch64', 'ppc64le', 's390x']
+        }
+    },
+    opensuse => {
+        Tumbleweed => {
+            released_registry => sub { "registry.opensuse.org/opensuse/tumbleweed" },
+            totest_registry   => sub {
+                "registry.opensuse.org/" . get_opensuse_registry_prefix . "opensuse/tumbleweed";
+            },
+            available_arch => ['x86_64', 'aarch64', 'ppc64le', 's390x', 'arm']
+        },
+        '15.0' => {
+            released_registry => sub { "registry.opensuse.org/opensuse/leap:15.0" },
+            totest_registry   => sub {
+                "registry.opensuse.org/opensuse/leap/15.0/images/totest/images/opensuse/leap:15.0";
+            },
+            available_arch => ['x86_64']
+        },
+        '15.1' => {
+            released_registry => sub { "registry.opensuse.org/opensuse/leap:15.1" },
+            totest_registry   => sub {
+                if (check_var('ARCH', 'x86_64')) {
+                    "registry.opensuse.org/opensuse/leap/15.1/images/totest/containers/opensuse/leap:15.1";
+                } elsif (check_var('ARCH', 'arm')) {
+                    "registry.opensuse.org/opensuse/leap/15.1/arm/images/totest/containers/opensuse/leap:15.1";
+                } else {
+                    "";
+                }
+
+            },
+            available_arch => ['x86_64', 'arm']
+        },
+        '15.2' => {
+            released_registry => sub { "registry.opensuse.org/opensuse/leap:15.2" },
+            totest_registry   => sub {
+                if (check_var('ARCH', 'x86_64')) {
```

Comment:
Using `check_var()` inhere is bad, as it would not use the given parameter to `get_suse_container_urls()`

Reply:
i think that s ok!!! 
we want `x86_64` container images on `x86_64` hosts.
or.. missing smth?

Reply:
no, I think you don't. Still I woundn't do that, as this apply only for `ARCH` but not for `VERSION` or `FLAVOR` which are agruments as well.
And maybe there will be a helper host running on `x86_64` which connects to a `arm` one via `ssh` and installing there some containers... now it belongs to the one who create the job what `ARCH` means... HOST/SUT what ever.
I think it is just more clean to not use it here, as we already got `arch` once on the argument handling.

Reply:
*this change is ready for review *
i replace the `check_var` with a conditional check of the passing `$arch`